components/link_checker/Cargo.toml

```diff
@@ -6,3 +6,6 @@ authors = ["Vincent Prouillet <prouillet.vincent@gmail.com>"]
 [dependencies]
 reqwest = "0.9"
 lazy_static = "1"
+scraper = "0.10"
```

Comment:
Is there a more lightweight scraper? This seems like it would add quite a few new dependencies.

Would it be OK to just do a simple `string.contains("id='the-id'") | string.contains(#"id="the-id'"#)`?